src/pudl/analysis/pudl_models.py

```diff
@@ -3,12 +3,10 @@
 import os
 
 import pandas as pd
-import pyarrow as pa
 from dagster import AssetsDefinition, asset
-from deltalake import DeltaTable
```

Comment:
If `deltalake` is no longer being directly imported anywhere then we should remove it from the requirements in `pyproject.toml`.